Sujata Singhal/FLASK/routing/FunctionsBasicII.py

```diff
@@ -0,0 +1,29 @@
+#Countdown - Create a function that accepts a number as an input.  Return a new array that counts down by one, from the number (as arrays 'zero'th element) down to 0 (as the last element).  For example countDown(5) should return [5,4,3,2,1,0].
+
+
+def countdown(num):
+	for x in range(num,0,-1):
```

Comment:
since the required output is an empty array, you need create an empty array arr=[] before the for loop replace print(x) on line 6 by arr.append(x)